test/disposable/SettableDisposable-test.js

```diff
@@ -14,7 +14,7 @@ describe('SettableDisposable', function () {
 
     var x = d.dispose()
 
-    expect(x).toBe(data)
+    eq(x, data)
```

Comment:
Unfortunately, it's not obvious from the [eq docs](https://github.com/briancavalier/assert#eq--a--a--a), but the intent is for the parameter ordering to be `expected, actual`.  It's more obvious when you partially apply it in assertions like: `promise.then(eq(expected))`.  Not a huge deal, but I'd like to go with `expected, actual` wherever possible.

I feel like `is` may be the right assertion here.  For example, this test could pass if the implementation of `SettableDisposable.prototype.dispose` were changed to simple `return {}`, since `eq({}, {}) === true`.  IOW, it is possible for `d.dispose` to conjure up a value that will pass the test, but it's not possible for it to conjure up a reference that will *unless* it's returned by the wrapped disposable.  Does that make sense?